css-layout-classes/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>CSS Layout Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h1>Choose your character!</h1>
+          </header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="Link character" />
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>
+            Super Smash Bros. Ultimate received universal acclaim from both
+            critics and players, with some critics calling it the best
+            installment in the series. It received praise for its large amount
+            of content and fine- tuning of exisiting Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="Yoshi" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-fourth">
+          <img src="images/kirby.png" alt="Kirby" />
+        </div>
+        <div class="column-fourth">
+          <img src="images/fox.png" alt="Fox" />
+        </div>
+        <div class="column-fourth">
+          <img src="images/pikachu.png" alt="Pikachu" />
+        </div>
+        <div class="column-fourth">
+          <img src="images/ness.png" alt="Pokemon" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-two">
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two game genres, but also includes
+            additional spin-off titles of various genres.
+          </p>
+
+          <p>
+            The games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
+        <div class="column-two">
+          <img src="images/dk.png" alt="donkey kong" />
+        </div>
```

Comment:
This shouldn't be a `column-two`. It doesn't take up the same size as the other column. Your text should be a `column-two-thirds` and this should be a `column-third`